pyproject.toml

```diff
@@ -51,6 +52,7 @@ nox = "^2023.4.22"
 python-dotenv = "^1.0.1"
 
 [tool.pytest.ini_options]
+addopts="-n=auto"
```

Comment:
`addopts = "-n=auto --asyncio-mode=auto"`

You could remove all `@pytest.mark.asyncio` and replace `@pytest_asyncio.fixture` with `@pytest.fixture` when this option is enabled.

Reply:
Good idea, fixed